librz/core/cmd_debug.c

```diff
@@ -1300,8 +1300,8 @@ static void cmd_debug_modules(RzCore *core, int mode) { // "dmm"
 			pj_end(pj);
 		} break;
 		case ':':
-		case '*':
-			if (mode == '*' || (mode == ':' && addr >= map->addr && addr < map->addr_end)) {
+		case 'RZ_OUTPUT_MODE_RIZIN':
```

Comment:
this change doesn't make sense. the command requires to be splitted.

Reply:
I am sorry but I haven't understood what you mean by "command should be split". Do you mean I should have a separate case for `mode == ':'`? Also, I noticed that I have accidentally put `RZ_OUTPUT_MODE_RIZIN` in single quotes. I'll change that as well.